apps/aehttpserver/lib/aehttpserver/web/channels/notify.ex

```diff
@@ -13,14 +14,16 @@ defmodule Aehttpserver.Web.Notify do
 
   def broadcast_new_block_added_to_chain_and_new_mined_tx(block) do
     Enum.each(block.txs, fn(tx) ->
-      Aehttpserver.Web.Endpoint.broadcast!("room:notifications", "new_mined_tx_everyone", %{"body" => Serialization.tx(tx, :serialize)})
-      if tx.data.from_acc != nil do
-        Aehttpserver.Web.Endpoint.broadcast!("room:notifications", "new_mined_tx:" <> Base.encode16(tx.data.from_acc), %{"body" => Serialization.tx(tx, :serialize)})
+      if match?(%SpendTx{}, tx.data) do
```

Comment:
I think we should also broadcast Oracle txs.

Reply:
yes, we should @d-velev